### DISCUSSION TITLE
fmiSetReal is not working in my case?

### DISCUSSION THREAD

**Discussion by Amuri44** [ThummeTo/FMI.jl#86] [Q&A] [answered]
I have an FMU (32bit) model, I found that the `fmiSetReal` is not working. Is there a way to know the reason?

```

**Comment by ThummeTo** [comment, marked as answer]
Hm, this might be an issue, because we defined `fmi2Real` hard as `Float64`. So the 32-bit FMU is using `Float32` I guess... this should be easy to fix, I will open an issue for you. 

Best regards!
